lib/xebow/excalibur/keyboard.ex

```diff
@@ -0,0 +1,284 @@
+require Logger
+
+defmodule Xebow.Excalibur.Keyboard do
```

Comment:
:information_source: (optional)

If this were in the namespace `Excalibur.Keyboard`, and `lib/excalibur`, would that help decouple it from the Xebow core code?

Reply:
Yeah, that might be a good idea...